composer.json

```diff
@@ -19,7 +19,7 @@
     "require": {
         "php":                       ">=5.3.3",
         "behat/behat":               "^3.4.3",
-        "symfony/framework-bundle":  "~2.0|~3.0|~4.0"
+        "symfony/framework-bundle":  "~2.0|~3.0|~4.0|~5.0"
```

Comment:
As Symfony versions `2.x` and `<3.4` are now out of support (https://symfony.com/releases) wouldn't it be better to drop their support now ?

```json
"require": {
    "symfony/framework-bundle": "~3.4|~4.2|~5.0"
}
```

Reply:
Good idea, will do in another PR.